podium-gateway/src/test/resources/config/application.yml

```diff
@@ -20,6 +20,18 @@ eureka:
         appname: podiumGateway
         instanceId: podiumGateway:${spring.application.instance_id:${random.value}}
 
+ribbon:
```

Comment:
Should be added to the [bbmri-podium-config](https://github.com/thehyve/bbmri-podium-config) as well!